<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GEyeTracker</class>
 <widget class="QWidget" name="GEyeTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>49</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GorgonEye</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="mainLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QFrame" name="viewFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QPushButton" name="startBtn">
         <property name="text">
          <string>Start</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopBtn">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="gazeBtn">
         <property name="text">
          <string>Gaze</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="viewSpacer">
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
